### DISCUSSION TITLE
Updating underlying syncthing core

### DISCUSSION THREAD

**Discussion by chadj** [MobiusSync/MobiusSync#45] [General]
I'm a happy paying customer here.  I was just wondering how often the team here plans on updating the underlying syncthing code base to be in line with the latest release?  1.17 has some neat new features that it would be nice to take advantage of.

**Comment by Petemir** [comment]
As a considering-to-pay future-customer I am also interested on this one :).

**Reply by Petemir** [reply]
[fyi it seems to be in the plans](https://twitter.com/MobiusSync/status/1406675346079064066)

**Comment by SimonPickup** [comment]
Update to Syncthing 1.18.x core is currently in TestFlight - see #56
